standard/README.md

```diff
@@ -370,7 +370,7 @@
     - [§11.17.2](expressions.md#11172-ambiguities-in-query-expressions)  Ambiguities in query expressions
     - [§11.17.3](expressions.md#11173-query-expression-translation)  Query expression translation
       - [§11.17.3.1](expressions.md#111731-general)  General
-      - [§11.17.3.2](expressions.md#111732-select-and-group--by-clauses-with-continuations)  select and group ... by clauses with continuations
+      - [§11.17.3.2](expressions.md#111732-select-and-group-…-by-clauses-with-continuations)  select and group … by clauses with continuations
```

Comment:
That's going to be interesting... I've no idea how browsers will handle URLs with an ellipsis in. I don't mind trying it though...